HttpProtocolException.md

```diff
@@ -0,0 +1,29 @@
+# Detect HTTP/2.0 and HTTP/3.0 protocol errors
+
+HTTP2/2 and HTTP/3 defines protocol-level error codes in [RFC 7540 section 7](https://www.rfc-editor.org/rfc/rfc7540#section-7) and [RFC 9114 section 8.1](https://www.rfc-editor.org/rfc/rfc9114.html#section-8.1) (not to be confused with HTTP status codes!). Although such low-level error information is unimportant for most `HttpClient` users, it might be helpful in advanced HTTP/2 or HTTP/3 scenarios, notably grpc-dotnet, where distinguishing protocol errors is vital to implement client retries.
+
+This is now possible by filtering for [`HttpProtocolException`](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpprotocolexception) when calling `HttpClient`:
```

Comment:
It feels like one more sentence is missing before that one, that would connect the first paragraph with the examples. It could be something like:
"The information about protocol-level error codes is now available in `HttpProtocolException.ErrorCode` property. `HttpProtocolException` would be either an inner exception of `HttpRequestException` (when calling `HttpClient`) or thrown directly (when reading from `HttpContent`'s response stream)"